tests/test-aws-lambda.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
were you able to run this script locally?  If so, could you add a new recipe to the `justfile` too, showing how it can be run/tested locally?  thx

Reply:
Ooh, even better:

```bash
 ¶ git diff .github/files/lambda.yaml
@@ -17,3 +17,6 @@ Resources:
         - Ref: MartinLayer
       CodeUri: lambda-function/
       Handler: config.yaml
+      Events:
+        ApiEvent:
+          Type: HttpApi
 ¶ sam local start-api --warm-containers eager
```

```bash
curl http://127.0.0.1:3000
```

Function URL is the best choice for running on AWS, but for local emulation we can configure an API Gateway HTTP API, and then send it HTTP requests instead of having to use `aws lambda invoke` with specially crafted event objects. Good for a followup.

Reply:
@jleedev would it make sense to create an issue to track it? Won't remember otherwise.